src/sage/rings/number_field/number_field.py

```diff
@@ -2148,27 +2159,25 @@ def random_element(self, num_bound=None, den_bound=None,
 
         INPUT:
 
-        - ``num_bound`` - Bound on numerator of the coefficients of
-                          the resulting element
+        - ``num_bound`` -- Bound on numerator of the coefficients of
+          the resulting element
 
-        - ``den_bound`` - Bound on denominators of the coefficients
-                          of the resulting element
+        - ``den_bound`` -- Bound on denominators of the coefficients
+          of the resulting element
 
-        - ``integral_coefficients`` (default: ``False``) - If ``True``, then
-                          the resulting element will have integral
-                          coefficients. This option overrides any
-                          value of `den_bound`.
+        - ``integral_coefficients`` (default: ``False``) --- If ``True``, then
+          the resulting element will have integral
+          coefficients. This option overrides any
+          value of ``den_bound``.
 
```

Comment:
`--` 

I think also that the default is better to be placed after `--`.

Reply:
I agree